pkg/build/controller/strategy/docker.go

```diff
@@ -81,6 +81,10 @@ func (bs *DockerBuildStrategy) CreateBuildPod(build *buildv1.Build, includeAddit
 							Name:      "buildworkdir",
 							MountPath: buildutil.BuildWorkDirMount,
 						},
+						{
+							Name:      "buildcachedir",
+							MountPath: "/var/lib/containers/cache/",
+						},
```

Comment:
gonna need to fix unit tests:
https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/sti_test.go#L105-L134
https://github.com/openshift/origin/blob/master/pkg/build/controller/strategy/docker_test.go#L65-L97

Reply:
done